x-pack/plugins/profiling/common/functions.test.ts

```diff
@@ -17,6 +17,6 @@ describe('TopN function operations', () => {
     expect(topNFunctions.TopN.length).toEqual(5);
 
     const exclusiveCounts = topNFunctions.TopN.map((value) => value.CountExclusive);
-    expect(exclusiveCounts).toEqual([16, 9, 7, 5, 2]);
+    expect(exclusiveCounts).toEqual([16, 9, 9, 5, 1]);
```

Comment:
Because I didn't change the order of the frames in the `stackTraces` fixture. That means the leaf frame is now a different frame than before, thus the results are different.

Reply:
So I swapped the leaf and the top frames in the 3rd commit - just to verify that the former result is seen again.